jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/content/AsyncContent.java

```diff
@@ -77,53 +68,19 @@ public void write(boolean last, ByteBuffer byteBuffer, Callback callback)
         offer(chunk, callback);
     }
```

Comment:
Technically, the chunk should be released after `offer` returns. We can avoid doing it because `offer` is private and we know it always retains to store the chunk for later and we save a useless retain/release pair.

But that reason probably is worth commenting.